gcloud/rest/auth/__init__.py

```diff
@@ -1,7 +1,10 @@
 from pkg_resources import get_distribution
```

Comment:
Oops. The auth files are not supposed to be here. Please ignore everything from that point down.

Reply:
Sure thing -- we'll need to make sure these get removed before merge! We'll get the base auth PR in first.

Reply:
Yep - I'll rebase onto master once #34 is merged. Since the auth changes will be on master already, they will disappear from this PR.